src/js/components/DataTable/Resizer.js

```diff
@@ -141,28 +164,70 @@ const Resizer = ({ onResize, property, headerText, messages, headerId }) => {
   return (
     <Keyboard onLeft={onKeyDown} onRight={onKeyDown}>
       <StyledResizer
-        tabIndex={0}
         aria-label={
           width ? `${ariaLabel} ${Math.trunc(width)} pixels` : ariaLabel
         }
         onMouseEnter={() => setHover(true)}
         onMouseLeave={() => setHover(false)}
-        onMouseDown={onMouseDown}
-        onMouseMove={start !== undefined ? onMouseMove : undefined}
-        onMouseUp={start !== undefined ? onMouseUp : undefined}
         ref={ref}
-        pad={{ vertical: 'xsmall' }}
-        margin={{ right: `-${theme.global.edgeSize.small}` }}
         role="separator"
         aria-valuenow={width}
         aria-valuetext={
           width ? `${ariaLabel} ${Math.trunc(width)} pixels` : ariaLabel
         }
         aria-controls={headerId}
         aria-orientation="vertical"
+        onMouseDown={onResizeStart}
+        onMouseMove={start !== undefined ? onResizeMove : undefined}
+        onMouseUp={start !== undefined ? onResizeEnd : undefined}
+        onTouchStart={onResizeStart}
+        onTouchMove={start !== undefined ? onResizeMove : undefined}
+        onTouchEnd={start !== undefined ? onResizeEnd : undefined}
+        dropContent={
+          <Box direction="row" pad="xsmall">
+            <Button
+              aria-label={format({
+                id: 'dataTable.decrease',
+                values: { headerText },
+                messages,
+              })}
+              icon={<Subtract />}
+              onClick={() => {
+                if (thRef.current) {
+                  const element = thRef.current;
+                  const rect = element.getBoundingClientRect();
+                  const currentWidth = rect.width;
+                  const nextWidth = Math.max(STEP, currentWidth - STEP);
+                  setWidth(nextWidth);
+                  onResize(property, nextWidth);
+                }
+              }}
+              autoFocus
+            />
+            <Button
+              aria-label={format({
+                id: 'dataTable.increase',
+                values: { headerText },
+                messages,
+              })}
+              icon={<Add />}
+              onClick={() => {
```

Comment:
should these calls be in a `useCallback` ?

Reply:
Pulled them out to `useCallback` to optimize so that functions aren't recreated on every render